src/components/layers/LayersToggle.js

```diff
@@ -31,7 +31,7 @@ LayersToggle.propTypes = {
 export default connect(
     (state) => ({
         isOpen: state.ui.layersPanelOpen,
-        isDownload: state.download.downloadMode,
+        isDownload: !!state.download.downloadMode,
```

Comment:
Without this `isDownload` can sometimes be `undefined`, like when clicking the "Exit download" button. This caused a prop-types error. With this `!!` `isDownload` is always a boolean and the error is not triggered. Arguably, the fix would actually lie in the reducer that produces this state, but I just went for this simple solution.